data/translations/nv/accordion.json

```diff
@@ -0,0 +1,45 @@
+    "content_5_part_3_link": "bee 44h0zin7 bik11’ aho’doolkidgo bik’ih jizd1ago nijitin7g77",
+    "content_5_part_4": "t’11 1ko. N11sg00 b7na’7d7[kid %link%. ",
+    "content_5_part_4_link": "i’ii’n77[ biniiy4 ha’dilne’",
+
+    "title_6": "Dah Dinin11d00 Bik44’d00 Ha’d7l4ehgo",
+    "content_6_part_1": "Azh3 t’11 1h1n7 d00 n7zaadg00 77n7n1a ndi bik’ehgo an1’7’a’7g77 1niid7go 1d77l77[ j0 k44h0t’7n7gi [ahgo 1dzaa. N7l47 dropdown dabik1’7g77 n7n7[’9 11d00 nitsaago Hahoodzoo7g77 n11n1 [ahj8’ 47 doodago territory woly4h7g77 biih y7n7n1ago 47 bidid77lchi[ 1kwe’4 bik11’ b44sh nits7kees biyi’j8’ hazh’di’dool99[ 47 doodago naaltsoos haa nin1dahajeeh7d00 hazh’diilaa dool44[ 11d00 ajidoo’a[. Nitsaago Hahoodzo biih y7n7n1n7g77 b44sh nits7kees7 biyi’j8’ an7’j7’a’7g77 b1 ha’dil’98go 47 t’11 h2h7 ts’98[go h1 h11d7l’88h. K0t’4ego [ahgo 1t’4ego h11’d7dleehgo t’1adoole’4 hada’diln4’4 bee e’e’aah7 ni h11’d7’d77dl99[, n7l47 Ada’iin77[ 30 yisk3 t’11 bich’8’ [eh [ahda. Nitsaago Hahoodzo biih y7n7n1n7g77 da nibik’ehgo na’ab22s7 1niid7g77 1n7l44h danidoonii[ 47 doodago naaltsoos bik’ehgo n44ho’d0lzingo nanitin7g77 1niid7 n1 1n7dooln77[. %link%",
+    "content_6_part_1_link": "Bee an1’7’a’7 haayit’4h7g77 nitsaago Hahoodzo biih y7n7n1n7g77 y0keed doolee[.",
+    "sub_head_part_2": "N11n1 [ahj8’ dah dinin1ago 1kondi t’11 nitsaago Hahoodzo biyi’ nighan7g77 t’11 47 biyi’ d7n7n1ago",
+    "content_6_part_2": "Azh3 t’11 nitsaago Hahoodzo biyi’ k44hon7t’7n7g77 t’11 47 biyi’ n11n1 [ahj8’ nin7n4ehgo a[d0’ bee an1’7’a’7 1niid7go n1 1dooln77[ n77n7n1n7gi bik1a’go.",
+    "sub_head_part_3": "N11n1 [ahj8’ dah dinin1ago 1kondi nitsaago Hahoodzo [ahgo 1t’4h7g77 biih  y7n7n1ago",
+    "content_6_part_3": "Nitsaago Hahoodzo biih y7n7n1n7g77 1[ts4 h11’d7’d77dl99[ t’1adoo bee e’e’aah7, n7l47 ada’iin77[ 30 yisk3 t’11 bich’8’ [eh [ahda. H11’d7dl4h7g77 t’1adoo n1 b7ighah sil98’g00 n7l47 Ada’ii’n77[ baa hodoolzhish7g77 t’11 b7ts4edi 1k0d77n77[ d77 nitsaago Hahoodzo biyi’ yin7n1n7gi, 1kodzaago 47 dah dinin1n7d00 da sh99 t’11 1ko i’7’11d doolee[7g77 t’00 naaltsoos naa n7doolwo[go 1kot’4ego a’d77’a[ 47 doodago t’11 ni 1k== d77n1a[go a’d77’a[. W11shindoondi ats7l1dei sid1h7g77 b1 ada’ii’n7i[go 47 nitsaago Hahoodzo biyi’d00 dah dinin1n4e t’11 1adi a’d77’a[ danididoonii[ 47 bee ada’iin7[7 t’00 [a’ naa n7doolwo[ 47 doodago t’11 ni 1k== d77n1a[go a’d77’a[. $7 bik44’d66 7nda naaltsoos hadid77l77[ nitsaago Hahoodzo biih y7n7n1n7jigo.",
+    "sub_head_part_4": "T0nteel ts’7naag00 d7n7n1ago",
+    "content_6_part_4": "U.S. biyi’d00 dine’4 n7l47 t0nteel bil1ahdi nijin7n1ago 47 a’da’ii’n77[ biniy4 hazhdi’dool77[ d00 bee ada’ii’n7[7 [a’ j0ki’ 47 naaltsoos the %link%. ",
+    "content_6_part_4_link": "Federal Post Card Application (FPCA)",
+    "content_6_part_5": "woly4ego hach’8’ 1l’88h. Kwe’4 the %link% woly4ego d7n77[’88[ 47 hazh0’0 yee ni[ hodoolnih sil1otahdi nijigh1ago d00 t0nteel bil11hd66’ a’ji’1adgo.",
+    "content_6_part_5_link": "Federal Voting Assistance Program (FVAP)"
+}
```

Comment:
This file is no longer required. You have to migrate this content into the `/content/<language-code>/accordion` directory as the others are.

Reply:
How (and from where) are these .md files generated?